apps/condo/domains/acquiring/utils/serverSchema/feeDistribution.spec.js

```diff
@@ -74,8 +74,8 @@ const TEST_CASES = [
         ],
         cases: [
             { amount: 1, expected: { type: 'service', summa: 1.01, recipientSum: 1, explicitFee: 0.01 } },
-            { amount: 5125812.99, expected: { type: 'service', summa: 5187818.79, recipientSum: 5125812.99, explicitFee: 62005.8 } },
-            { amount: 1746.12, category: null, expected: { type: 'service', summa: 1767.24, recipientSum: 1746.12, explicitFee: 21.12 } },
+            { amount: 5125812.99, expected: { type: 'service', summa: 5187322.75, recipientSum: 5125812.99, explicitFee: 61509.76 } },
```

Comment:
Why field is called `summa` in condo module. (And why is near `recipientSum`? :) )

Reply:
Not sure why, since I'm not an author of this module. And looks like I can not rename quickly, since it used in bunch of modules. I guess we have to do some sort of refactoring for that

Reply:
Can you add a TODO for that?